src/inlineeditoruiview.js

```diff
@@ -35,7 +35,7 @@ export default class InlineEditorUIView extends EditorUIView {
 		 * @readonly
 		 * @member {module:ui/toolbar/toolbarview~ToolbarView}
 		 */
-		this.toolbar = new ToolbarView( locale );
+		this.toolbar = new ToolbarView( locale, { shouldGroupWhenFull: options.shouldToolbarGroupWhenFull } );
```

Comment:
[We usually put these in 2 lines as a rule of thumb for better readability](https://github.com/ckeditor/ckeditor5-editor-classic/blob/master/src/classiceditoruiview.js#L52-L54).

Reply:
👍